Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM amazonlinux:2017.03
+FROM amazonlinux:2018.03
+
+COPY requirements.txt quilt/requirements.txt
```

Comment:
I'd move this down, to just above `pip install` - otherwise, updating requirements will cause Docker to needlessly re-run "yum install".